File: EASSET.Web/Modules/MasterData/OPD/OPDForm.cs
using Serenity.ComponentModel;
using System;

namespace EASSET.MasterData.Forms;

[FormScript("MasterData.OPD")]
[BasedOnRow(typeof(OPDRow), CheckNames = true)]
public class OPDForm
{
    public string UnitKode { get; set; }
    public string UnitNama { get; set; }
    public string Npwp { get; set; }
    public short IsBlud { get; set; }
    public int NoUrut { get; set; }
    //public string ThAng { get; set; }
    [TextAreaEditor(Rows = 5, Cols = 10)]
    public string Alamat { get; set; }
    //public string Op { get; set; }
    //public DateTime Lu { get; set; }
    //public string Pc { get; set; }
    //public bool Dlt { get; set; }
}

File: EASSET.Web/Modules/ServerTypes/MasterData/OPDForm.ts
import { PrefixedContext, initFormType } from "@serenity-is/corelib";
import { StringEditor, IntegerEditor, TextAreaEditor } from "serenity.corelib";

export interface OPDForm {
    UnitKode: StringEditor;
    UnitNama: StringEditor;
    Npwp: StringEditor;
    IsBlud: IntegerEditor;
    NoUrut: IntegerEditor;
    Alamat: TextAreaEditor;
}

export class OPDForm extends PrefixedContext {
    static readonly formKey = 'MasterData.OPD';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!OPDForm.init)  {
            OPDForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;
            var w2 = TextAreaEditor;

            initFormType(OPDForm, [
                'UnitKode', w0,
                'UnitNama', w0,
                'Npwp', w0,
                'IsBlud', w1,
                'NoUrut', w1,
                'Alamat', w2
            ]);
        }
    }
}